R/pkg/R/DataFrame.R

```diff
@@ -936,7 +936,9 @@ setMethod("unique",
 
 #' Sample
 #'
-#' Return a sampled subset of this SparkDataFrame using a random seed.
+#' Return a sampled subset of this SparkDataFrame using a random seed. 
+#' Note that this is not guaranteed to provide exactly the fraction specified
```

Comment:
Yeah I see what you're saying but I think it might be a bit out of scope for this PR. What your goal is definitely makes sense as there should be a standard, but I don't think I'm degrading anything significantly. I will add the `:` character.